src/Demo/Logs.java

```diff
@@ -0,0 +1,33 @@
+package Demo;
+
+public class Logs {
+    private static Logs logs;
+
+    private Logs() {
+
+    } 
+
+
+  //error/info/debug
+
+  public static Logs getInstance(){
+
+    if(logs == null){
+        logs= new Logs();
+    }
+    return logs;
+}
+
+public void getError() {
```

Comment:
you usually log in this way `error("error message")` so, there is no much point on having getters here from my point of view.